packages/components/psammead-image-placeholder/src/index.jsx

```diff
@@ -5,14 +5,17 @@ import { BBC_BLOCKS } from '@bbc/psammead-assets/svgs';
 
 const ImagePlaceholder = styled.div`
   position: relative;
-  height: 0;
+  height: ${props => props.ratio}vw;
```

Comment:
`vw` doesn't seem to be supported in [opera mini](https://caniuse.com/#search=vw) are we sure using `height & vw` is a better solution than `padding-bottom & %`?

Reply:
![caption](https://user-images.githubusercontent.com/7791726/54840405-e45b4b00-4cc4-11e9-8f02-3ccaaff04ded.gif)

@dr3 found a bug with the `image-placeholder` logic when using `vw` it's easiest to replicate using browserstack. I think we should look at limiting the width of the wrapping div rather than the figure, this way we can persist with the `padding-bottom & %` which appears to be more consistent than `height & vw`